spec/features/user_cannot_access_admin_privileges_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+RSpec.describe 'Access', type: :feature do
+  create_category
+  create_brand
+  create_items
+  create_user
+  create_rental
+
+    feature 'user' do
```

Comment:
Inconsistent indentation detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.